jsk_tools/CMakeLists.txt

```diff
@@ -41,12 +32,36 @@ if (CATKIN_ENABLE_TESTING)
   roslint_python(src/test_stdout.py)
   roslint_add_test()
   find_package(rostest REQUIRED)
+  macro(jsk_tools_add_rostest file)
+    list(APPEND jsk_tools_test_files ${file})
+    add_rostest(${file})
+  endmacro()
   if("$ENV{ROS_DISTRO}" STRGREATER "hydro")
     # FIXME: jsk_tools/test_topic_published.py does not work on hydro travis/jenkins
     # https://github.com/jsk-ros-pkg/jsk_common/pull/1293#issuecomment-164158260
-    add_rostest(test/test_topic_published.launch)
+    jsk_tools_add_rostest(test/test_topic_published.launch)
+    # FIXME: jsk_tools/sanity_lib.py does not work on hydro travis/jenkins
+    # https://github.com/jsk-ros-pkg/jsk_common/pull/1405#issuecomment-225218625
+    jsk_tools_add_rostest(test/python/sanity_lib.test)
   endif()
-  add_rostest(test/test_stdout.test)
+  jsk_tools_add_rostest(test/test_stdout.test)
```

Comment:
yes.

2016年6月13日月曜日、Kei Okada<notifications@github.com>さんは書きました:

> In jsk_tools/CMakeLists.txt
> <https://github.com/jsk-ros-pkg/jsk_common/pull/1405#discussion_r66737440>
> :
>
> >    endif()
> > -  add_rostest(test/test_stdout.test)
> > +  jsk_tools_add_rostest(test/test_stdout.test)
>
> i see, you're using this at ->
> https://github.com/jsk-ros-pkg/jsk_common/pull/1405/files#diff-1c7ebe416e617a508fa2bfbef49120a7R53
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jsk-ros-pkg/jsk_common/pull/1405/files/7446221f3859503dad950af776ab9e161c4e198b#r66737440>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEHFk6e_sxDybSM0jDzVfesIWvwrTCQzks5qLNTZgaJpZM4Iu0zy>
> .
>


-- 
和田 健太郎 / Kentaro Wada
http://wkentaro.com